src/frontend/Pretty_printing.ml

```diff
@@ -436,8 +438,9 @@ and pp_statement ppf ({stmt= s_content; smeta= {loc}} as ss : untyped_statement)
     =
   match s_content with
   | Assignment {assign_lhs= l; assign_op= assop; assign_rhs= e} ->
-      pf ppf "@[<h>%a %a %a;@]" pp_lvalue l pp_assignmentoperator assop
-        pp_expression e
+      pf ppf "@[<h>%a %a%a %a;@]" pp_lvalue l
+        (pp_comments_spacing get_comments_until_separator)
```

Comment:
Assignment operators aren't separators so `comment 6` in `tuples./pretty.expected` ends up on the wrong side of it.